lib/file_codec.py

```diff
@@ -134,6 +135,22 @@ def MatroskaFrameInfo(encodedfile):
   return frameinfo
 
 
+def FfmpegFrameInfo(encodedfile):
+  # Uses the ffprobe tool to give frame info.
+  commandline = '%s -loglevel warning -show_frames -of json %s' % (
+      encoder.Tool('ffprobe'), encodedfile)
+  ffprobeinfo = subprocess.check_output(commandline, shell=True)
+  probeinfo = json.loads(ffprobeinfo)
+  pos = 0
+  frameinfo = []
+  for frame in probeinfo['frames']:
+    if pos != 0:
+      frameinfo.append({'size': 8*(int(frame['pkt_pos']) - pos)})
```

Comment:
This will become easier to read with
curr_pkt_pos = int(frame['pkt_pos'])
frameinfo.append({'size': 8 * (curr_pkt_pos - previous_pkt_pos)})